pages/glossary/visual-context.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Visual context
+key: visual-context
+---
+
+The visual context of some piece of information (_e.g._ an element) is everything that is visually or logically located near it. Only [visible](#visible) information may be part of the visual context.
```

Comment:
Defnining "near" is difficult. Could it mean all information on the web page? Or all information in the viewport? Again this carries with it a slew of other issues...

Reply:
Yes, "near" is annoying… The problem with viewport or similar is that "on screen" info is that this can change by zooming or scrolling and I don't want, say, a header to disappear from context just because it's been scrolled 1 line out of screen…

Reply:
There are a bunch of notes clarifying the meaning of "near", both in that definition and in the global definition of "context". Do you think we need more?

Reply:
I agree with you. I do not have anything to add.